### DISCUSSION TITLE
LLDPd and discovery of tagged VLANs

### DISCUSSION THREAD

**Discussion by fcolista** [lldpd/lldpd#473] [Q&A]
Is there any way, with LLDPd and a switch that speaks LLDP, to know which VLANs are tagged on the port where the server is connected?
Thanks!

.: Francesco

**Comment by vincentbernat** [comment]
Yes, this is part of the 802.1. If the remote switch has it enabled, you should see VLANs with `lldpcli show neighbor details`.

**Comment by ciroiriarte** [comment]
Hello!, digging the grave of this post to expand on the topic (and maybe mark it as Answered?). 

I was about to open a bug report because I only saw 5 VLANs out of 17 on a JunOS switch.

It turns out switches can limit the publication of VLANs. My findings so far:

- Cisco requires explicit configuration
- JunOS defaults to 5 VLAN max, can be extended. Limit depends on MTU.

My notes: https://github.com/ciroiriarte/nic-xray#:~:text=Switch%20configuration%3A
